incubator/hnc/e2e/tests/delete_crd_test.go

```diff
@@ -0,0 +1,44 @@
+package test
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = Describe("Delete-anchor-crd", func() {
+	const (
+		hncYamlEnvVar = "HNC_REPAIR"
+	)
+	
+	AfterEach(func() {
+		// clean up
+		nses := []string{
+			"delete-crd-parent",
+			"delete-crd-child",
+		}
+		// Remove all possible objections HNC might have to deleting a network. Make sure it 
+		// has cascading deletion so we can delete any of its subnamespace descendants, and 
+		// make sure that it's not a subnamespace itself so we can delete it directly.
+		for _, ns := range nses {
+			mustRun("kubectl hns set", ns, "-a")
+			tryRun("kubectl annotate ns", ns, "hnc.x-k8s.io/subnamespaceOf-")
+			mustRun("kubectl delete ns", ns)
```

Comment:
I think these should all by tryRun - what happens if the namespace wasn't successfully created? Then they'll all fail.

Reply:
done